docker/docker-compose.yml

```diff
@@ -27,3 +27,12 @@ services:
       SCHEDULE_TOPICS: schedules
     ports:
     - "9095:9095"
+
+  prometheus:
+    image: prom/prometheus:v1.7.2
```

Comment:
Quite an old verison now (Sept 2017), maybe worth using a newer one?

Reply:
upgraded to 2.25.0 <- latest.